simulation/traffic_simulator/src/vehicle_model/sim_model_delay_steer_acc_geared.cpp

```diff
@@ -120,17 +118,29 @@ float64_t SimModelDelaySteerAccGeared::calcVelocityWithGear(
     gear == GearCommand::DRIVE_16 || gear == GearCommand::DRIVE_17 ||
     gear == GearCommand::DRIVE_18 || gear == GearCommand::LOW || gear == GearCommand::LOW_2) {
     if (state(IDX::VX) < 0.0) {
-      return 0.0;
+      state(IDX::VX) = 0.0;
+      state(IDX::X) = prev_state(IDX::X);
+      state(IDX::Y) = prev_state(IDX::Y);
+      state(IDX::YAW) = prev_state(IDX::YAW);
     }
   } else if (gear == GearCommand::REVERSE || gear == GearCommand::REVERSE_2) {
     if (state(IDX::VX) > 0.0) {
-      return 0.0;
+      state(IDX::VX) = 0.0;
+      state(IDX::X) = prev_state(IDX::X);
+      state(IDX::Y) = prev_state(IDX::Y);
+      state(IDX::YAW) = prev_state(IDX::YAW);
     }
   } else if (gear == GearCommand::PARK) {
-    return 0.0;
+    state(IDX::VX) = 0.0;
+    state(IDX::X) = prev_state(IDX::X);
+    state(IDX::Y) = prev_state(IDX::Y);
+    state(IDX::YAW) = prev_state(IDX::YAW);
   } else {
-    return 0.0;
+    state(IDX::VX) = 0.0;
+    state(IDX::X) = prev_state(IDX::X);
+    state(IDX::Y) = prev_state(IDX::Y);
+    state(IDX::YAW) = prev_state(IDX::YAW);
   }
 
-  return state(IDX::VX);
-}
+  state(IDX::ACCX) = (state(IDX::VX) - prev_state(IDX::VX)) / std::max(dt, 1.0e-5);
+}
```

Comment:
Add newline to the end of the file using a suitable text editor.

Reply:
Fixed in https://github.com/tier4/scenario_simulator_v2/pull/707/commits/2ec83d51597ce0bfe63f09d5fb0deddb5789cc86

Reply:
It hasn't been fixed.

Reply:
Sorry fixed in https://github.com/tier4/scenario_simulator_v2/pull/707/commits/dde2ff3a6c16a24f4693e2b98da8b1a0186840c7